code/modules/mob/living/carbon/brain/brain_item.dm

```diff
@@ -59,7 +59,6 @@
 /obj/item/organ/brain/removed(var/mob/living/user)
 
 	if(!owner) return ..() // Probably a redundant removal; just bail
-	name = "[owner.real_name]'s brain"

```

Comment:
Um...doesn't this mean literally every single brain is going to end up being just "brain"? If so, that's horrendous.